src/legacy/ui/public/agg_types/buckets/_inline_comp_wrapper.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
```

Comment:
Should this be a .jsx file?

Reply:
I haven't seen we ever use the `jsx` file extension. Is there any benefit to use it?
Do not think it would be useful to have only `jsx` file, which uses 3 line of jsx code.
The same thoughts has Dan Abramov https://github.com/airbnb/javascript/pull/985#issuecomment-239145468